src/java.base/share/classes/java/io/Console.java

```diff
@@ -145,6 +146,66 @@ public Reader reader() {
         throw newUnsupportedOperationException();
     }
 
+    /**
+     * Writes a string representation of the specified object to this console's
+     * output stream, terminates the line and then flushes the console.
```

Comment:
Should this specify if the line termination will be platform dependent character(s) or independent of the platform?

Reply:
That's a good question. I think it should. That `println` method is not specified in terms of `printf` or `format`. Thus, we cannot reduce `println` to, say, `printf("%s%n", obj)`, leaning on `Formatter`'s definition of `%n`:

    'n'	line separator	The result is the platform-specific line separator

@stuart-marks, any thoughts on wording? Mind you, if we add any such wording, we'll have to update CSR too.

Reply:
Well `Formatter` defers to `System.lineSeparator()` so I think we should refer to that. Not sure that needs to be repeated in `java.io.IO`.